From 65058aab3f986f9048c5f045945fe3a3034da056 Mon Sep 17 00:00:00 2001
From: Daniele Forsi <daniele@forsi.it>
Date: Mon, 28 Dec 2009 23:00:35 +0100
Subject: [PATCH] Make bluetooth_type_to_string() return a translated string

Use the _() macro which calls gettext() instead of the N_() macro that only
marks strings as translatable.

https://bugzilla.gnome.org/show_bug.cgi?id=605631
---
 lib/bluetooth-client.c |   30 +++++++++++++++---------------
 1 files changed, 15 insertions(+), 15 deletions(-)

diff --git a/lib/bluetooth-client.c b/lib/bluetooth-client.c
index 6dd7a6e..d55f53f 100644
--- a/lib/bluetooth-client.c
+++ b/lib/bluetooth-client.c
@@ -112,36 +112,36 @@ const gchar *bluetooth_type_to_string(BluetoothType type)
 {
 	switch (type) {
 	case BLUETOOTH_TYPE_ANY:
-		return N_("All types");
+		return _("All types");
 	case BLUETOOTH_TYPE_PHONE:
-		return N_("Phone");
+		return _("Phone");
 	case BLUETOOTH_TYPE_MODEM:
-		return N_("Modem");
+		return _("Modem");
 	case BLUETOOTH_TYPE_COMPUTER:
-		return N_("Computer");
+		return _("Computer");
 	case BLUETOOTH_TYPE_NETWORK:
-		return N_("Network");
+		return _("Network");
 	case BLUETOOTH_TYPE_HEADSET:
 		/* translators: a hands-free headset, a combination of a single speaker with a microphone */
-		return N_("Headset");
+		return _("Headset");
 	case BLUETOOTH_TYPE_HEADPHONES:
-		return N_("Headphones");
+		return _("Headphones");
 	case BLUETOOTH_TYPE_OTHER_AUDIO:
-		return N_("Audio device");
+		return _("Audio device");
 	case BLUETOOTH_TYPE_KEYBOARD:
-		return N_("Keyboard");
+		return _("Keyboard");
 	case BLUETOOTH_TYPE_MOUSE:
-		return N_("Mouse");
+		return _("Mouse");
 	case BLUETOOTH_TYPE_CAMERA:
-		return N_("Camera");
+		return _("Camera");
 	case BLUETOOTH_TYPE_PRINTER:
-		return N_("Printer");
+		return _("Printer");
 	case BLUETOOTH_TYPE_JOYPAD:
-		return N_("Joypad");
+		return _("Joypad");
 	case BLUETOOTH_TYPE_TABLET:
-		return N_("Tablet");
+		return _("Tablet");
 	default:
-		return N_("Unknown");
+		return _("Unknown");
 	}
 }
 
-- 
1.7.0.1

